charts/kafka-connect/values.yaml

```diff
@@ -0,0 +1,280 @@
+  ## @param service.clusterIP Service Cluster IP
+  ## e.g.:
+  ## clusterIP: None
+  ##
+  clusterIP: ""
+  ## @param service.externalTrafficPolicy Service external traffic policy
+  ## ref https://kubernetes.io/docs/tasks/access-application-cluster/create-external-load-balancer/#preserving-the-client-source-ip
+  ##
+  externalTrafficPolicy: Cluster
+  ## @param service.annotations Provide any additional annotations which may be required.
+  ## This can be used to set the LoadBalancer service type to internal only.
+  ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#internal-load-balancer
+  ##
+  annotations: {}
+  ## @param service.sessionAffinity Session Affinity for Kubernetes service, can be "None" or "ClientIP"
+  ## If "ClientIP", consecutive client requests will be directed to the same Pod
+  ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#virtual-ips-and-service-proxies
+  ##
+  sessionAffinity: None
+  ## @param service.sessionAffinityConfig Additional settings for the sessionAffinity
+  ## sessionAffinityConfig:
+  ##   clientIP:
+  ##     timeoutSeconds: 300
+  ##
+  sessionAffinityConfig: {}
+
+## The Kafka Connect properties
+## ref: https://docs.confluent.io/current/connect/userguide.html#configuring-workers
+configurationOverrides:
+  ##  @param configurationOverrides.plugin.path the plugin path in the file system
+  "plugin.path": "/usr/share/java,/usr/share/confluent-hub-components"
+  ##  @param configurationOverrides.key.converter the serializer for keys
+  "key.converter": "io.confluent.connect.avro.AvroConverter"
+  ##  @param configurationOverrides.value.converter the serializer for values
+  "value.converter": "io.confluent.connect.avro.AvroConverter"
+  ##  @param configurationOverrides.key.converter.schemas.enable enable schemas for key converter
+  "key.converter.schemas.enable": "false"
+  ##  @param configurationOverrides.value.converter.schemas.enable enable schemas for value converter
+  "value.converter.schemas.enable": "false"
+  ##  @param configurationOverrides.internal.key.converter internal key converter
+  "internal.key.converter": "org.apache.kafka.connect.json.JsonConverter"
+  ##  @param configurationOverrides.internal.value.converter internal value converter
+  "internal.value.converter": "org.apache.kafka.connect.json.JsonConverter"
+  ##  @param configurationOverrides.config.storage.replication.factor config storage replication factor
+  "config.storage.replication.factor": "3"
+  ##  @param configurationOverrides.offset.storage.replication.factor offset storage replication factor
+  "offset.storage.replication.factor": "3"
+  ##  @param configurationOverrides.status.storage.replication.factor status storage replication factor
+  "status.storage.replication.factor": "3"
+
+## @param heapOptions Kafka Connect JVM Heap Option
+heapOptions: "-Xms512M -Xmx512M"
+
+## Additional env variables
+## @param customEnv Additional env variables
+customEnv: {}
```

Comment:
This should be array. If we keep this as dict then we won't be able to use `valueFrom` in envs in deployment

Reply:
done